configuration/reports/reportdescriptors/dataexports/sql/msppReport.sql

```diff
@@ -609,5 +812,94 @@ SELECT SUM(child_under_1_n) "CHILD_UNDER_1_N",SUM(child_under_1_s) "CHILD_UNDER_
             @malaria_test_microscopique_oval'OVAL',
             @malaria_test_microscopique_malariae 'MALARIAE',
             @malaria_test_microscopique_falciparum 'FALCIPARUM',
-            @malaria_test_microscopique_total 'MALARIAN_TEST_MICRO'
+            @malaria_test_microscopique_total 'MALARIAN_TEST_MICRO',
+

```

Comment:
my SQL client did not like these blank lines in the middle of the query.  I needed to remove them to get this to run.  Not sure if it will cause a problem in OpenMRS.